draft-ietf-ppm-dap.md

```diff
@@ -1335,10 +1331,10 @@ Once all the report shares have been initialized, the Leader creates an
 opaque BatchID[32];
 
 struct {
-  QueryType query_type;
-  select (PartialBatchSelector.query_type) {
+  BatchMode batch_mode;
+  select (PartialBatchSelector.batch_mode) {
     case time_interval: Empty;
-    case fixed_size: BatchID batch_id;
+    case leader_selected: BatchID batch_id;
   };
 } PartialBatchSelector;
```

Comment:
While at it: Do we still like the term "partial batch selector"?

Reply:
I do not like "partial batch selector" at all, but I have struggled to come up with a better name (e.g. more descriptive, without being too long). Suggestions for a better name are very welcome IMO